code/obj/machinery/deep_fryer.dm

```diff
@@ -196,8 +196,9 @@
 
 			thing.reagents.add_reagent("grease", 50)
 			fryholder.desc = "A heavily fried...something.  Who can tell anymore?"
-		if (istype(thing, /obj/item/reagent_containers/food/snacks))
-			fryholder.food_effects += thing:food_effects
+		if (istype(src.fryitem, /obj/item/reagent_containers/food/snacks))
+			fryholder.food_effects += fryitem:food_effects
```

Comment:
colon: bad

Reply:
Replaced colon with proper typecasting.

Also removed a couple colons from further up the file while I was there. Appears to work just fine.